crypto/bio/bss_conn.c

```diff
@@ -546,9 +546,9 @@ static long conn_ctrl(BIO *b, int cmd, long num, void *ptr)
             BIO_set_ktls_flag(b, num);
         break;
     case BIO_CTRL_GET_KTLS_SEND:
-        return BIO_should_ktls_flag(b, 1);
+        return BIO_should_ktls_flag(b, 1) ? 1 : 0;
```

Comment:
We usually use `!= 0` in such case instead of the ternary `?` operator.

Reply:
Ok, I was copying from BIO_CTRL_EOF a few lines above which does the ternary.  I'm happy to change it to != 0 instead though as that is what I normally do myself.

Reply:
You can fix that place as well to make it uniform, if you want.